README.md

```diff
@@ -32,7 +32,7 @@ Here is [quick demo](https://asciinema.org/a/380928).
 ## Features
 
 * Simple script based installer to deploy OpenShift (4.5 onwards) cluster on PowerVS leveraging Infrastructure as Code (IaC) pattern
-* Supports multiple (64-bit) x86 platforms including Linux, Windows & Mac OSX
+* Supports multiple (64-bit) x86 platforms including Linux, Windows & Mac OSX, also Linux for ppc64le
```

Comment:
Just change it to "Supports multiple (64-bit) platforms including Linux(x86_64/ppc64le), Windows & Mac OSX"